src/collections/_documentation/accounts/pricing.md

```diff
@@ -50,22 +48,262 @@ You can set a maximum monthly on-demand bill amount by setting an on-demand spen
 
 #### Rate limits
 
-Business and Enterprise plans have the option to configure rate limits for each project that let you specify the number of events and time interval for rate limiting.
+Team and Business plans have the option to configure rate limits for each project that let you specify the number of events and time interval for rate limiting.
 
 ## Managing your event volume
 
 If your projects have a high volume of events, you can control how many errors Sentry receives by reviewing Sentry's guide to [Manage Your Event Stream](/accounts/quotas/manage-event-stream-guide/).
 
 ## FAQ
+{% comment %}
+The on-demand chart needs to sit as the first question to enable proper linking from the sign up page. 
+{% endcomment %}
+
+`Is there detailed information for each plan?`
+
+Yes, the following table details costs for errors, transactions, and attachments.
+
+<table>
```

Comment:
You can do markdown tables like here: https://github.com/getsentry/sentry-docs/blob/53212163781891be57715b292ddbbe7911dd76d2/src/collections/_documentation/platforms/python/pyspark.md#L51-L55

Reply:
Yes, I've used both. Tables are always tricky. Do we prefer one over the other?